shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/LeafFormat.kt

```diff
@@ -88,16 +96,16 @@ sealed class LeafFormat {
     }
 
     class BranchedBuilder {
-        private val branches = mutableListOf<Branched.Branch>()
+        private val branchRows = mutableListOf<Branched.BranchRow>()
         var secondaryAlert: UpcomingFormat.SecondaryAlert? = null
 
         fun branch(headsign: String, format: UpcomingFormat) = branch(null, headsign, format)
```

Comment:
It seems like these helper functions for building the `Branched` should also be `branchRow`.